### PR TITLE
Make token revocation immediately consistent

### PR DESCRIPTION
**Pull request by rjkip**
Also some misc. maintenance.

qwerty

### CODE REVIEW

**Comment by rjkip** [comment]
Ping @DRvanR

File: src/Surfnet/StepupMiddleware/CommandHandlingBundle/Identity/CommandHandler/IdentityCommandHandler.php [hunk 1]
```diff
@@ -47,11 +50,36 @@ class IdentityCommandHandler extends CommandHandler
...
+     * @param Connection $middlewareConnection
+     * @param Connection $gatewayConnection
      */
-    public function __construct(IdentityRepository $repository)
-    {
+    public function __construct(
+        IdentityRepository $repository,
+        BufferedEventBus $eventBus,
+        Connection $middlewareConnection,
+        Connection $gatewayConnection
```
Comment (DRvanR) [inline src/Surfnet/StepupMiddleware/CommandHandlingBundle/Identity/CommandHandler/IdentityCommandHandler.php]: Could split it out to two CommandHandlers, no need to do, just sayin' :wink:

File: src/Surfnet/StepupMiddleware/CommandHandlingBundle/Tests/Identity/CommandHandler/CommandHandlerTest.php [hunk 2]
```diff
@@ -19,6 +19,7 @@
 namespace Surfnet\StepupMiddleware\CommandHandlingBundle\Tests\Identity\CommandHandler;
 
 use Broadway\CommandHandling\Testing\CommandHandlerScenarioTestCase;
+use Mockery as m;
```
Comment (DRvanR) [inline src/Surfnet/StepupMiddleware/CommandHandlingBundle/Tests/Identity/CommandHandler/CommandHandlerTest.php]: Still needed?

**Comment by DRvanR** [comment]
Just a little check for an added namespace, :+1: though

**Pull request merged**